import-export-cli/box/resources/kubernetes_resources/api_cr.yaml

```diff
@@ -4,3 +4,5 @@ metadata:
   name:
 spec:
   swaggerConfigMapName:
+  paramsValues:
+  certsValues:
```

Comment:
We can keep the new line at the end of the line? WDYT? 
It is better to keep, as an example if someone uses CAT command the last line will get omitted.

Reply:
Fixed with 60ffa8adb24316d4d7b9fcbdc61476b31af7f787